elasticsearch/includes/classes/class-elasticsearch.php

```diff
@@ -50,6 +50,9 @@ protected function setup_constants() {
 	protected function setup_hooks() {
 		add_filter( 'ep_index_name', [ $this, 'filter__ep_index_name' ], PHP_INT_MAX, 3 ); // We want to enforce the naming, so run this really late.
 
+		// Override default per page value set in elasticsearch/elasticpress/includes/classes/Indexable.php
+		add_filter( 'ep_bulk_items_per_page', [ $this, 'filter__ep_bulk_items_per_page' ], PHP_INT_MAX, 0 );
```

Comment:
```suggestion
		add_filter( 'ep_bulk_items_per_page', [ $this, 'filter__ep_bulk_items_per_page' ], PHP_INT_MAX );
```

The "args count" param can be omitted here - it's only needed if your callback is expecting more than 1 argument.